crates/fuel-core/src/database.rs

```diff
@@ -255,9 +257,19 @@ where
     }
 
     #[cfg(feature = "rocksdb")]
-    pub fn rocksdb_temp(rewind_policy: StateRewindPolicy) -> Result<Self> {
-        let db = RocksDb::<Historical<Description>>::default_open_temp(None)?;
-        let historical_db = HistoricalRocksDB::new(db, rewind_policy)?;
+    pub fn rocksdb_temp(
```

Comment:
We could use database config here as well

Reply:
Done